be/src/exec/pipeline/scan/scan_operator.h

```diff
@@ -166,8 +166,8 @@ class ScanOperatorFactory : public SourceOperatorFactory {
     Status prepare(RuntimeState* state) override;
     void close(RuntimeState* state) override;
 
-    bool need_local_shuffle() const override { return _need_local_shuffle; }
```

Comment:
why rename need_local_shuffle to could_local_shuffle, it sounds a little reluctant.

Reply:
`need_local_shuffle=false` actually means DO NOT add local shuffle, so `could` is better than `need`